docs/internals/protocol/messages.rst

```diff
@@ -280,6 +280,10 @@ Known headers:
 * 0xFF04 ``ALLOW_CAPABILITIES``: ``uint64`` -- optional bitmask of
   capabilities allowed for this query.  See RFC1004_ for more information.
 
+* 0xFF05 ``IMPLICIT_SHAPEIDS`` -- "true" by default.
```

Comment:
* I think it's better to call this `OMIT_OBJECTIDS`.

Reply:
To invert the default value? Maybe `EXPLICIT_OBJECTIDS` is better? "omit" sounds like they will forcefully be omitted always.

Reply:
> Maybe EXPLICIT_OBJECTIDS is better?

Yeah, much better, good idea.